src/main/scala/uk/gov/hmrc/govukfrontend/views/implicits/ImplicitsSupport.scala

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.govukfrontend.views.implicits
+
+import play.api.data.{Field, FormError}
+import play.api.i18n.Messages
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.{HtmlContent, Text}
+import uk.gov.hmrc.govukfrontend.views.viewmodels.errormessage.ErrorMessage
+
+trait ImplicitsSupport[T] {
```

Comment:
Has this trait moved package?

Reply:
@matthewmascord Yes it has, however the only place it's used other than this repo is in play-frontend-hmrc.